### PR TITLE
Removing ES6 constructs from pages used in WebDriver tests

### PR DESCRIPTION
**Pull request by jimevans**
Internet Explorer doesn't support ES6 constructs like fat arrow functions.
So that the WebDriver tests can be executed against IE, the pages loaded
in the browser by the tests shouldn't include such constructs.

### CODE REVIEW

File: webdriver/tests/element_click/bubbling.py [hunk 1]
```diff
@@ -67,17 +68,19 @@ def test_spin_event_loop(session):
...
+            setTimeout(function() { window.delayedClicks.push(target); }, 100);
           });
         }
         </script>
         """)
     three, two, one = session.find.css("div")
     one.click()
 
-    delayed_clicks = session.execute_script("return window.delayedClicks")
+    delayed_clicks = session.execute_script("return window.delayedClicks;")
```
Comment (andreastt) [inline webdriver/tests/element_click/bubbling.py]: FWIW the semicolon shouldn’t be necessary as the injected script is wrapped in a function, and semicolons aren’t required in JS.

File: webdriver/tests/element_click/bubbling.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 from tests.support.asserts import assert_success
 from tests.support.inline import inline
-
+from time import sleep
```
Comment (andreastt) [inline webdriver/tests/element_click/bubbling.py]: Unused?

**Review by andreastt — CHANGES_REQUESTED** [review]

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/24653)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/344362311)

File: webdriver/tests/element_click/bubbling.py (hunk 2 above)
Comment (jimevans) [reply webdriver/tests/element_click/bubbling.py]: D'oh! Fixed in another commit.

**Review by andreastt — APPROVED** [review]

File: webdriver/tests/element_click/bubbling.py (hunk 1 above)
Comment (jimevans) [reply webdriver/tests/element_click/bubbling.py]: Reverted to previous syntax, without semicolons.

**Pull request merged**